src/skydb.ts

```diff
@@ -90,14 +90,14 @@ export async function getJSON(
 
   if (!(data["_data"] && data["_v"])) {
     // Legacy data prior to v4, return as-is.
-    return { data, skylink };
+    return { data, dataLink: skylink };
   }
 
   const actualData = data["_data"];
   if (typeof actualData !== "object" || data === null) {
     throw new Error(`File data '_data' for the entry at data key '${dataKey}' is not JSON.`);
   }
-  return { data: actualData as Record<string, unknown>, skylink };
+  return { data: actualData as Record<string, unknown>, dataLink: skylink };
```

Comment:
Is this type not equal to `JsonData`? Would re-use that. This also applies to line :85.

Reply:
Good catched